samples/04_gis_analysts_data_scientists/traffic-light-detection-on-oriented-imagery.ipynb

```diff
@@ -161,24 +161,7 @@
    "outputs": [
```

Comment:
<ul><li><code>'color': [255,0.0]</code> --&gt; <code>'color': [255,0,0]</code> for red points</li><li><code>'size': '8px'</code> --&gt; <code>'size': 8</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
thanks for catching that!